perllib/FixMyStreet/Cobrand/Default.pm

```diff
@@ -851,16 +851,16 @@ sub get_report_stats { return 0; }
 sub get_body_sender {
     my ( $self, $body, $category ) = @_;
 
+    # look up via category
+    my $contact = $body->result_source->schema->resultset("Contact")->search( { body_id => $body->id, category => $category } )->first;
```

Comment:
If we're doing this all the time now, note that it is also then looked up again by Email and Open311 SendReports, so we should probably think of a way it only needs to be done once.